app/controllers/patients_controller.rb

```diff
@@ -57,6 +57,11 @@ def destroy
     redirect_to patients_path
   end
 
+  def search_patient
```

Comment:
Let's follow the REST convention, only public `index`, `create`,` edit`, `update`, `show` and `delete` methods allowed in controllers